pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/SchemaRegistryServiceImpl.java

```diff
@@ -270,6 +270,9 @@ private void checkCompatible(SchemaAndMetadata existingSchema, SchemaData newSch
     @Override
     public CompletableFuture<Void> checkConsumerCompatibility(String schemaId, SchemaData schemaData,
                                                               SchemaCompatibilityStrategy strategy) {
+        if (SchemaCompatibilityStrategy.ALWAYS_COMPATIBLE.equals(strategy)) {
```

Comment:
this is an enum, please use "=="

Reply:
![image](https://user-images.githubusercontent.com/9758905/141108064-34ea301a-c73d-4e3c-a252-abea9b3234eb.png)

What is the difference ?